step/src/main/webapp/profile.jsp

```diff
@@ -25,18 +32,87 @@ limitations under the License.
   <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style_profile.css">
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
-    <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>My Profile</title>
   </head>
   <body>
     <nav>
         <a href="<%= logoutURL %>"> Log Out </a>
     </nav>
     <h1>Friend Matching Plus </h1>
-    <p>Here is your profile! </p>
 
     <% if (!userService.isUserLoggedIn()) {
         response.sendRedirect("index.jsp");   
     } %>
+
+    <%
+    String id = userService.getCurrentUser().getUserId();
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query =
+        new Query("UserInfo")
+            .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+    PreparedQuery results = datastore.prepare(query);
+    Entity entity = results.asSingleEntity();
+    %>
+
+    <h2> Your Portfolio </h2>
+    <div class="container">
+        <div class="sub-container" id="list-selection">
+            <div id="profile-pic"> <img src="avatar.png" alt="Profile Picture"> </div>
+            <div id="navbar-selection"> 
+                <a href="#personal-container"> Personal Information </a>
+                <a href="#questionaire-container"> Questionaire </a> 
+                <a href="#matches-container">Your Matches </a>
+            </div>
+        </div>
+        <div class="sub-container" id="profile-info">
+            <h3> <a href="infoForm.jsp"> Personal Infomation </a> </h3>
+            <div class="personal-container" id="personal-container">
+                <div class="personal-item">
+                    <div class="item-label"> First Name </div> 
+                    <div class="item-info"> <%= (String) entity.getProperty("firstName")%> </div>
+                </div>
+                <div class="personal-item">
+                    <div class="item-label">Last Name </div> 
+                    <div class="item-info"><%= (String) entity.getProperty("lastName")%> </div> 
+                </div>
+                <div class="personal-item">
+                    <div class="item-label">Email </div> 
+                    <div class="item-info"><%= (String) entity.getProperty("email")%> </div> 
+                </div>
+                <div class="personal-item">
+                    <div class="item-label">Date of Birth </div> 
+                    <div class="item-info"><%= String.valueOf(entity.getProperty("monthBirth"))%>/<%= String.valueOf(entity.getProperty("dayBirth"))%>/<%= String.valueOf(entity.getProperty("yearBirth"))%> </div> 
```

Comment:
nit: if it won't break things, can we split this line into several lines?

Reply:
It will. The line is the date which the format should be mm/dd/yy. I think they should go along with each other.